circle.yml

```diff
@@ -30,6 +30,8 @@ dependencies:
 
 test:
   override:
+    # Check whether the committer forgot to run `go generate`.
+    - docker run cockroachdb/cockroach-dev shell "go generate ./... && git ls-files --modified --deleted | diff /dev/null -"
```

Comment:
I think the git ls-files call should also include --others to find new/untracked files.